cpu/lpc2387/include/lpc2387.h

```diff
@@ -151,4 +151,4 @@ extern "C" {
 }
 #endif
 
-#endif // __LPC2387_H
+#endif /* __LPC2387_H */
```

Comment:
Mh, this one is still (or again?) prefixed with `__`. Please fix and squash immediately.